core/src/plugins/kubernetes/config.ts

```diff
@@ -902,15 +903,15 @@ export const portForwardsSchema = () =>
 
 export const runPodSpecWhitelistDescription = () => runPodSpecIncludeFields.map((f) => `* \`${f}\``).join("\n")
 
-export const runCacheResultSchema = () =>
+export const runCacheResultSchema = (kind: ActionKind) =>
   cacheResultSchema().description(
     dedent`
-Set to false if you don't want the Runs's result to be cached. Use this if the Run needs to be run any time your project (or one or more of the Run's dependants) is deployed. Otherwise the Run is only re-run when its version changes, or when you run \`garden run\`.
+Set to false if you don't want the ${kind} action result to be cached. Use this if the ${kind} needs to be run any time your project (or one or more of the ${kind}'s dependants) is deployed. Otherwise the ${kind} is only re-run when its version changes, or when you run \`garden run\`.
```

Comment:
My suggestion also applied to every other occurrence of Kind in these sentences:
```suggestion
Set to false if you don't want the ${kind} action result to be cached. Use this if the ${kind} action needs to be run any time your project (or one or more of the ${kind} action's dependants) is deployed. Otherwise the ${kind} action is only re-run when its version changes, or when you run \`garden run\`.
```